### ISSUE TITLE
Hi, its us again.

### ISSUE THREAD

**Issue by knaub-am-bau** [kovan-testnet/faucet#77]
Hi, its us again.
We are in need for more KEth.

The reason for that is that we have more testers in the alpha. And like I explained in the previous issue, it is quite expansive what we are doing.

Thank you very much.

_Originally posted by @knaub-am-bau in https://github.com/kovan-testnet/faucet/issues/76#issuecomment-467391708_

**Comment by knaub-am-bau** [comment]
Hi. Is there any problem with the faucet.

We dont wnt to bother but we need some mor keth.

Thank you verry much.

**Comment by Tbaut** [comment]
I'm sorry but you'll have to be more specific regarding what you do. The account still has 200+ ETH in its balance, and it looks like you are sending 10KETH to each tester. I can't think of a use case where this is a justified amount. I've [already mentioned this](https://github.com/kovan-testnet/faucet/issues/61#issuecomment-444116931) in the past. This represent a threat to the network.

**Comment by knaub-am-bau** [comment]
Hi. We don't want to threat the network. We have another solution, so we don't need more Keth at the moment. I will contact you again, if this doesn't work out, and then provide you with further information. But thanks for your answer.

**Comment by knaub-am-bau** [comment]
Hi, 
we made it successfully through the alpha.
We improved our use of Keth.

We managend to get along with the amount you  provided us.

Now we have to ask you again for KEth.
Because our beta is running and we see the end of our fuel coming soon.

I also have to update you on our website, it changed its domain name.
Now it is ash.finance and it is way more informative then the old one.
We still running on the MLN protocol.
Our eth address is also still the same

If you have any questions, pls let me know.
Thanks.
